packages/slice-machine/src/components/Kbd/Kbd.css.ts

```diff
@@ -0,0 +1,14 @@
+import { style } from "@vanilla-extract/css";
+import { colors, sprinkles } from "@prismicio/editor-ui";
+
+export const root = style([
+  sprinkles({
+    paddingInline: 4,
+    color: colors.grey11,
+  }),
+  {
+    border: "1px solid #E4E2E4",
```

Comment:
Ask @benjammartin if we can have a color from Design System for both color here.